x-pack/plugins/ml/public/application/jobs/new_job/pages/components/summary_step/summary.tsx

```diff
@@ -92,13 +92,13 @@ export const SummaryStep: FC<StepProps> = ({ setCurrentStep, isCurrentStep }) =>
       advancedStartDatafeed(jobCreator, navigateToPath);
     } catch (error) {
       // catch and display all job creation errors
-      const { toasts } = notifications;
-      toasts.addDanger({
-        title: i18n.translate('xpack.ml.newJob.wizard.summaryStep.createJobError', {
+      const { displayErrorToast } = toastNotificationServiceProvider(notifications.toasts);
+      displayErrorToast(
+        error,
+        i18n.translate('xpack.ml.newJob.wizard.summaryStep.createJobError', {
```

Comment:
Not sure we meant for the same i18n id here but if we are using the same block for both places I wonder if we should consolidate them in one function.

Reply:
these ids have always been the same, which surprises me, but i guess it doesn't complain if the ids are the same and the text is the same.
I'll remove the duplication.

Reply:
Removed in https://github.com/elastic/kibana/pull/76743/commits/9027b2aad6a7e788410528121288096a8e69b44c